spec/features/user/edit_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
```

Comment:
This test is written well. It verifies that emails are used properly in verifying a user when they register.